### ISSUE TITLE
Updates for v3

### ISSUE THREAD

**Issue by snipe** [snipe/snipe-it#2066]
Hi all,

As you may have noticed, v3.0-alpha2 is now out. We'd love it if you guys can kick the tires, either as a new installation or an upgrade (but use a copy of your production database - this is still in alpha.)

What's New:
This link is constantly being updated, so please check back in often:
https://snipe-it.readme.io/blog/v3-is-coming

Upgrading:
https://snipe-it.readme.io/docs/upgrading

Installing (new):
https://snipe-it.readme.io/docs/getting-started

MAKE SURE YOU INSTALL FROM THE v3 BRANCH:
https://github.com/snipe/snipe-it/tree/v3

### Known Issues
- The install.sh has not yet been updated. Do not try to use that to install v3.
- We're having a few issues with the top menu on responsive mobile. We're working on it, but would love it if any CSS ninjas could take a look as well. We've got our hands a little full trying to fix bugs just prior to the v3 official release.
- If you change the language (either in the app settings or your profile) and the language doesn't change, it might be that the language you've selected isn't fully translated yet. 

If you're a bilingual non-techie but still want to help, we'd love to get more of the new strings translated before v3: https://crowdin.com/project/snipe-it

When creating tickets for v3-alpha, please note that in your ticket subject. 

## Stuff We Need Tested
Obviously, a top to bottom test is ideal, but these are the bits we really want to beat up:
- Authentication and authorization - we rewrote that whole system, and want to make sure it's still solid.
- The new features listed at https://snipe-it.readme.io/blog/v3-is-coming
- Mobile views - (small bugs may not be a launch blocker for us, but we'd still like to know about them.)
- Label printing - we've made a lot of changes to this, so would like to make sure it's working well. (Also, if you have a particular set of labels that you've worked out the settings for, please share them.)
- If you have a 2D barcode scanner, please check that the barcodes on the labels are working :) You'll need to put your cursor focus on the top search bar and then scan. We're working on making that less clunky.

**Comment by Profecy** [comment]
So far I have tested some of the 2D and 3D Barcodes and they seem to be readable just fine and contain the asset ID. 

Tested 3D Codes: QR and Datamatrix
Tested 2D Codes: PDF417 and Code39

They print nicely with the settings below.

Also, I would like to share my settings for the labels I am currently using in the testing phase.

Label Printer: DYMO LabelWriter 250 Duo
Label type: 99012

Labels per page: 1
Label font size: I use 8 or 9 but that's personal taste.
Label dimensions (inches): 3.50394 w x 1.41732 h
Label spacing (inches): 0 w x 0 h
Page margins (inches): 0.02000 top 0.00000 bottom 0.00000 right 0.20000 left
Page dimensions (inches): 3.50394 w x 1.41732 h

In Chrome I have set the margins to "none". So no need to fiddle with them.

So far, v3 seems pretty solid. I like it a lot.

**Comment by snipe** [comment]
@Profecy thanks so much for the feedback and for sharing your settings!

**Comment by Profecy** [comment]
@snipe my pleasure! :+1:

**Comment by markjorissen** [comment]
Also testing v3, but we need to get the custom fields issue fixed (see my post about this error, which also appears in the online demo) before we can test it further (we are going to use it real-time). So far it seems very good. I really like the interface and features. Looks a lot better than GLPI or other programs. Hope to hear from you soon!

**Comment by vjandrea** [comment]
On a clean v3 install i can't get past the login screen, as far as i see there's a problem reading the user settings.

`[2016-06-03 17:50:37] local.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /<mypath>/vjandrea-snipe-it/app/Http/Controllers/Auth/AuthController.php:186
Stack trace:
#0 /<mypath>/vjandrea-snipe-it/app/Http/Controllers/Auth/AuthController.php(186): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Trying to get p...', '<mypath>...', 186, Array)
#1 [internal function]: App\Http\Controllers\Auth\AuthController->login()`

Full stack trace here: http://laravel.io/bin/1yDzm

**Comment by snipe** [comment]
@vjandrea Which v3? We've had a few different RCs and beta releases. (Also, next time please create a new issue for this, as it's unrelated to the original thread)

**Comment by vjandrea** [comment]
Sorry, i thought it was related to this: "Authentication and authorization - we rewrote that whole system, and want to make sure it's still solid." - anyway please discard, i'm opening a new thread with all the details.
